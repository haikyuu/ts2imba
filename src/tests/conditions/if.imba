import {build} from "../../index"
import * as imbac from 'imba/compiler'

let tsx-code = '''
if(e){
	let a =2
}else{
	let b = 2
}
'''

test 'if else basic 124' do
	const result = await build tsx-code
	expect(result.code).toMatchSnapshot()
	const out = imbac.compile(result.code, sourceId: 'sth')