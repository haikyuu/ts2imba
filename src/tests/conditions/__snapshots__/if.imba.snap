// Vitest Snapshot v1

exports[`if else basic 124 1`] = `
"if e
	let a = 2
else
	let b = 2
"
`;
